bluesky/examples.py

```diff
@@ -473,7 +473,8 @@ def stage(self):
         self._file_stem = short_uid()
         self._path_stem = os.path.join(self.save_path, self._file_stem)
```

Comment:
No, I mean that my adding 'root' to the insert you need to _not_ do this join as root is joined to the path in filestore and you will end up with `'/save/path/save/path/file/stem'`